packages/app/src/systems/Error/guards/ReportErrorGuard.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+import { Outlet } from 'react-router-dom';
+
+import { useReportError } from '../hooks';
+
+import { store } from '~/store';
+
+export function ReportErrorGuard() {
+  const { reportErrorSilently, hasErrorsToReport } = useReportError();
+
+  useEffect(() => {
+    if (hasErrorsToReport) {
+      if (reportErrorSilently) {
```

Comment:
should this if statement be outside of the `hasErrorsToReport` statement?  I think the name `reportsErrorsSilently` is confusing to me because even if this boolean is true it still opens the popup right?

Reply:
No, if the `reportErrorsSilently` is `true`, it doesn't open the pop up.

The report silently feature was built to support an error reporting API endpoint. 
We don't have that so I'm removing the feature.